metricflow/model/model_validator.py

```diff
@@ -51,26 +52,30 @@ class ModelValidator:
         ReservedKeywordsRule(),
     )
 
-    def __init__(self, rules: Sequence[ModelValidationRule] = DEFAULT_RULES) -> None:
+    def __init__(self, rules: Sequence[ModelValidationRule] = DEFAULT_RULES, max_workers: int = 4) -> None:
```

Comment:
I thought we were defaulting to 1?

Reply:
Huh this comment was out of date as soon as it was made 🤔 After dropping the commit that added the conditional logic (which was also the commit that made the default 1), I added a separate commit that set the default as 1. I wonder if github somehow showed you the changes without that commit?